app/webpacker/components/ScrambleMatcher/Groups.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { Button, Header } from 'semantic-ui-react';
+import I18n from '../../lib/i18n';
+import ScrambleMatch from './ScrambleMatch';
+
+const scrambleToName = (scramble) => `Scramble ${scramble.scramble_number} (${scramble.scramble_string})`;
+
+export default function Groups({
+  scrambleSetCount,
+  scrambleSets,
+  dispatchMatchState,
+}) {
+  const [selectedGroupNumber, setSelectedGroupNumber] = useState(null);
+
+  const availableEventIds = useMemo(
+    () => Array.from({ length: scrambleSetCount }, (e, i) => i),
+    [scrambleSetCount],
+  );
+
+  const onGroupDragCompleted = useCallback(
+    (fromIndex, toIndex) => dispatchMatchState({
+      type: 'moveScrambleInSet',
+      setNumber: selectedGroupNumber,
+      fromIndex,
+      toIndex,
+    }),
+    [dispatchMatchState, selectedGroupNumber],
+  );
+
+  const groupScrambleToName = useCallback(
+    (idx) => `Attempt ${idx + 1}`,
```

Comment:
I thought we are using `.scramble_number` here now

Reply:
I am using it in line 6 above, which is the function that gets passed to label the draggable scramble cell itself.

The one you've picked here is the left-hand side "definition" column, which stays static irrespective of the scramble you're dragging.